idx/memory/memory.go

```diff
@@ -63,6 +71,14 @@ var (
 	findCacheBackoffTime         = time.Minute
 )
 
+func init() {
+	// instantiate strats for the interning layer/object store
```

Comment:
`strats`

Reply:
Yeah, and this also introduces the same problem as described in #1327